hwtracer/src/perf/mod.rs

```diff
@@ -6,24 +6,31 @@ pub(crate) mod collect;
 const PERF_DFLT_DATA_BUFSIZE: size_t = 4096; // 16MiB
 /// The default perf aux buffer size, measured in pages. Must be a power of 2.
 const PERF_DFLT_AUX_BUFSIZE: size_t = 16; // 64KiB
+/// The size of the final memory returned, measured in bytes. There are no constraints on size or
```

Comment:
On the lines above the comments should have appended "(for a 4096-byte page)"? Maybe I'm being too pedantic?

Reply:
This stuff is sufficiently broken that I don't think the extra pedantry really helps TBH.